src/extension.ts

```diff
@@ -11,15 +11,18 @@ import { initializeSpotify } from "./managers/PlaylistDataManager";
 import { displayReadmeIfNotExists } from './DataController';
 import { getUser } from './managers/UserStatusManager';
 import { clearSpotifyAccessToken } from './managers/SpotifyManager';
+import { LocalStorageManager } from './managers/LocalStorageManager';
 
 let currentColorKind: number = undefined;
+let storageManager: LocalStorageManager | undefined = undefined;
 
 export function deactivate(ctx: ExtensionContext) {
   clearWebsocketClient();
   clearSpotifyAccessToken();
 }
 
 export async function activate(ctx: ExtensionContext) {
+	storageManager = LocalStorageManager.getInstance(ctx);
```

Comment:
indentation inconsistent?

Reply:
yeah, updated it with eslint and ran fix-it